src/Foundation/Base/code/HCA.Foundation.Base.csproj

```diff
@@ -525,6 +537,10 @@
       <Project>{0044b0ca-abe2-4365-b1b2-1e347993b4aa}</Project>
       <Name>HCA.Foundation.GlassMapper</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\ReactJss\code\HCA.Foundation.ReactJss.csproj">
```

Comment:
Now you can't use, for example, logging service in `HCA.Foundation.ReactJss`. Remove this dependency and find a better place for processor.